### DISCUSSION TITLE
Is there any plan to remove chartmuseum from harbor? chartmuseum will not be supported in the future?

### DISCUSSION THREAD

**Discussion by zhhray** [goharbor/harbor#20981] [Q&A] [answered]
Is there any plan to remove chartmuseum from harbor? chartmuseum will not be supported in the future?
I'm not sure about this goal. If there is a related plan, could you please tell the reason and schedule?

**Comment by MinerYang** [comment, marked as answer]
Please refer to https://github.com/goharbor/harbor/discussions/15057 for more details

**Reply by zhhray** [reply]
> Please refer to #15057 for more details

Thank you very mach. I have noticed that chartmuseum is deprecated in v3.8.
